src/core/qgis.h

```diff
@@ -68,6 +68,25 @@ class CORE_EXPORT Qgis
      */
     static QString releaseName();
 
+    /**
+     * QGIS release type
+     * \since QGIS 3.16
+     */
+    enum ReleaseType
+    {
+      Latest = 0, //!< Latest release or point release
```

Comment:
"Latest" seems a bit misleading to me -- it conveys that the user is running the most recent release, but instead this value will remain "latest" for a 3.14 install even after 3.20 is released :wink:

Maybe "Regular" or "Normal" is a better name?

Reply:
Yeah, I before thought naming it "Stable" but I think "Regular" is a better word for latest release.